### ISSUE TITLE
getSnapshotUrl("1972-01-01") give false positives in linux

### ISSUE THREAD

**Issue by andrie** [RevolutionAnalytics/checkpoint#192]
Test results:

```
1. Failure (at test-0-snapshots.R#36): snapshot functions return correct results 
getSnapshotUrl("1972-01-01") no warnings given
```
